Makefile

```diff
@@ -153,6 +153,10 @@ test-init:
 	go test -count=1 -short -v -coverprofile cover.out ./ecs-init/...
 	go tool cover -func cover.out > coverprofile-init.out
 
+test-ecs-agent:
```

Comment:
nit (non-blocking): Should we consider just running these as part of the `test` target? I think wanting to run tests in both agent/ and ecs-agent/ would be more common than wanting to run them separately. In automatic CI workflows, this doesn't really matter, but for those who manually run these targets, running two commands seems less preferable than just running a single one. This also follows the existing pattern of the `gogenerate` targets (i.e., `make gogenerate` runs for agent/ and ecs-agent/ and there is no `make gogenerate-ecs-agent`).

Reply:
Yeah I like that idea. I will update.